kofta/public/locales/nl/translation.json

```
@@ -0,0 +1,222 @@
+			"currentRoom": "huidige deelname in:",
+			"startPrivateRoom": "samen een privé kamer beginnen"
+		},
+		"followList": {},
+		"home": {
+			"createRoom": "Kamer aanmaken"
+		},
+		"inviteList": {
+			"roomGone": "kamer ontbreekt, ga terug",
+			"shareRoomLink": "link delen met kamer",
+			"inviteFollowers": "Je kan je volgers uitnodigen die online zijn:",
+			"whenFollowersOnline": "Wanneer je volgers online zijn zullen zij hier staan."
+		},
+		"login": {
+			"headerText": "Wij brengen gesprekken naar de maan 🚀",
+			"featureText_1": "Donker Thema",
+			"featureText_2": "Openbare registraties",
+			"featureText_3": "Platformoverstijgende ondersteuning",
+			"featureText_4": "Open Source",
+			"featureText_5": "Tekst Berichten",
+			"featureText_6": "Aangedreven door Doge",
+			"loginGithub": "inloggen met GitHub",
+			"loginTwitter": "inloggen met Twitter",
+			"createTestUser": "test gebruiker aanmaken"
+		},
+		"myProfile": {
+			"logout": "uitloggen",
+			"probablyLoading": "waarschijnlijk aan het laden...",
+			"voiceSettings": "naar gesprek instellingen",
+			"soundSettings": "naar geluids-instellingen",
+			"deleteAccount": "account verwijderen"
+		},
+		"notFound": {
+			"whoopsError": "Oops! Deze pagina is kwijtgeraakt in een conversatie.",
+			"goHomeMessage": "Geen zorgen. Jij kan het",
+			"goHomeLinkText": "naar thuispagina"
+		},
+		"room": {
+			"speakers": "Luidsprekers",
+			"requestingToSpeak": "Aanvragen om te mogen spreken",
+			"listeners": "Luisteraars"
+		},
+		"searchUser": {
+			"search": "zoeken..."
+		},
+		"soundEffectSettings": {
+			"header": "Geluiden"
+		},
+		"viewUser": {
+			"editProfile": "profiel bewerken",
+			"followsYou": "volgt jou",
+			"followers": "volgers",
+			"following": "volgt"
+		},
+		"voiceSettings": {
+			"header": "Stem instellingen",
+			"mic": "microfoon:",
+			"permissionError": "geen microfoon gevonden, je hebt hem niet aangesloten of de website heeft geen permissie om je te horen.",
+			"refresh": "microfoon lijst verversen",
+			"volume": "volume:"
+		}
+	},
+	"components": {
+		"avatar": {},
+		"backBar": {},
+		"blockedFromRoomUsers": {
+			"header": "Verbannen gebruikers",
+			"unban": "Verbanning ongedaan maken",
+			"noBans": "niemand is nog verbannen"
+		},
+		"bottomVoiceControl": {
+			"leaveCurrentRoomBtn": "Huidige kamer verlaten",
+			"confirmLeaveRoom": "Weet je het zeker dat je deze kamer wilt verlaten?",
+			"leave": "Verlaten",
+			"inviteUsersToRoomBtn": "Gebruikers uitnodigen",
+			"invite": "Uitnodigen",
+			"toggleMuteMicBtn": "Microfoon dempen schakelen",
+			"mute": "Demp",
+			"unmute": "Dempen opheffen",
+			"makeRoomPublicBtn": "Kamer publiek maken!",
+			"settings": "Instellingen",
+			"speaker": "Luidspreker",
+			"listener": "Luisteraar"
+		},
+		"deviceNotSupported": {
+			"notSupported": "Je toestel is momenteel niet ondersteund. Maak een",
+			"linkText": "kwestie aan op GitHub",
```

```suggestion
			"linkText": "issue aan op GitHub",
```
Het heten altijd issues op Github